<dec f='tensorflow/tensorflow/core/lib/core/error_codes.pb.h' l='70' type='14'/>
<use f='tensorflow/tensorflow/core/lib/core/errors.h' l='107' u='r' c='_ZN10tensorflow6errors11UnavailableEDpT_'/>
<use f='tensorflow/tensorflow/core/lib/core/errors.h' l='107' u='r' c='_ZN10tensorflow6errors13IsUnavailableERKNS_6StatusE'/>
<use f='tensorflow/tensorflow/c/tf_status_helper.cc' l='71' c='_ZN10tensorflow25Set_TF_Status_from_StatusEP9TF_StatusRKNS_6StatusE'/>
<use f='tensorflow/tensorflow/core/debug/debug_io_utils.cc' l='446' u='r' c='_ZN10tensorflow7DebugIO18PublishDebugTensorERKNS_12DebugNodeKeyERKNS_6TensorEyRKNS_3gtl10ArraySliceISsEEb'/>
<use f='tensorflow/tensorflow/core/lib/core/status.cc' l='96' c='_ZNK10tensorflow6Status8ToStringEv'/>
<use f='tensorflow/tensorflow/core/platform/posix/error.cc' l='138' u='r' c='_ZN10tensorflow11ErrnoToCodeEi'/>
